src/plugins/WebOfTrust/util/SynchronousDelayedBackgroundJob.java

```diff
@@ -0,0 +1,231 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * Implementation of delayed background jobs where {@link #triggerExecution} blocks until the
+ * background job has finished, as if the background job was performed synchronously. This
+ * class is intended to be used in unit tests, and is likely not very useful for general purposes.
+ *
+ * @author bertm
+ */
+public class SynchronousDelayedBackgroundJob implements DelayedBackgroundJob {
```

Comment:
I don't see any reason to make this class final, given we don't rely on its exact semantics anywhere.